tests/framework/installer/ceph_manifests.go

```diff
@@ -1882,6 +1887,8 @@ spec:
   crashCollector:
     disable: false
   storage:
+    config:
+	  ` + crushRoot + `
```

Comment:
The yaml requires spaces instead of tabs. Looks like there is a tab here so it might fail.

Reply:
The tests don't make it easy to see what settings are enabled in the cluster CR. While you're in here, could you write the test cluster CR to the log? For example, on line 222 in ceph_installer.go, add this:
```
rookCluster := h.Manifests.GetRookCluster(settings)
logger.Info(rookCluster)
```